qbplatform/quote/models.py

```diff
@@ -13,19 +15,59 @@ class Tournament(models.Model):
 
     name = models.CharField(max_length=500)
 
+    def __str__(self):
+        return self.name
+
 
 class TournamentLocation(models.Model):
 
     tournament = models.ForeignKey(Tournament, on_delete=models.CASCADE)
     date = models.DateField()
     location = models.CharField(max_length=500, null=True)
 
+    def __str__(self):
+        return self.location + " site of tournament: " + str(self.tournament)
+
 
 class Packet(models.Model):
 
     name = models.CharField(max_length=500)
     tournament = models.ForeignKey(Tournament, on_delete=models.CASCADE)
 
+    def __str__(self):
+        return "Packet " + self.name + " in tournament: " + str(self.tournament)
+
+#   Model of a per-packet distribution
+#       has a one-to-many relationship with DistributionCategory
+class Distribution(models.Model):
+
+    num_tossups = models.PositiveSmallIntegerField(null=True, default=20)
+    num_bonuses = models.PositiveSmallIntegerField(null=True, default=20)
+
+    # a distribution is unique to a tournament/question set 
+    tournament = models.OneToOneField(Tournament, null=True, on_delete=models.CASCADE)
```

Comment:
I'm not sure that `OneToOneField` makes sense here. A distribution is logically independent of any specific tournament using it, and therefore many tournaments should be able to use the same distribution. I think it makes more sense to have `Distribution` be a foreign key on the `Tournament` object instead. And I would also add a `name` field so that each distribution can be named with some nice descriptive name.

Reply:
Made these changes in [ebfc59f](https://github.com/grapesmoker/qb-question-platform/pull/23/commits/ebfc59f352ab3300e977fed9aa28cbe2050af95d).